PolicyServerElasticsearchContainer/src/main/scripts/setup-database.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright 2015-2018 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+CONNECTION_URL=$(env | grep hibernate.connectionstring | awk -F'=' '{print $2}')
```

Comment:
this translation of getting the environment value and assign it to the variable is needed because  `hibernate.connectionstring`  has a 'dot' in it, we can not just use in the script. Other way is to change the property in the repo, but that has been referenced in too many places